crates/sui-core/src/epoch/committee_store.rs

```diff
@@ -15,8 +17,13 @@ use typed_store_derive::DBMapUtils;
 
 use sui_macros::nondeterministic;
 
-#[derive(DBMapUtils)]
 pub struct CommitteeStore {
+    tables: CommitteeStoreTables,
+    cache: RwLock<HashMap<EpochId, Committee>>,
```

Comment:
This is also ended up needed - without it we were deserializing committee on every call to CommitteeStore:: get_committee, and if we included PublicKey into committee serde, the performance suffered a lot